Lombiq.Hosting.MediaTheme/Recipes/_MediaTheme/Templates/Example.liquid

```diff
@@ -1,3 +1,6 @@
 <p>This is an example template hosted in Media Theme.</p>
 <p>Below is an image also hosted in Media Theme:</p>
 <img src="{{ '~/mediatheme/example.png' | href }}" alt="example" />
+
+{% assign favicon = '~/mediatheme/favicon.ico' | href %}
+{% link rel:"icon", type:"image/ico", sizes:"48x48", href:favicon, append_version:"true" %}
```

Comment:
A safer approach is to include this as an `img` as well, to make sure that the browser attempts to load it, and thus we'll get a 404 logged if it can't.